modules/cli-options/src/main/scala/scala/cli/commands/ScalacOptions.scala

```diff
@@ -33,29 +32,34 @@ object ScalacOptions {
     Set("-V", "-W", "-X", "-Y")
   private val scalacOptionsPrefixes =
     Set("-g", "-language", "-opt", "-P", "-target") ++ scalacOptionsPurePrefixes
+
+  /** This includes all the scalac options which disregard inputs and print a help and/or context
+    * message instead.
+    */
+  val ScalacPrintOptions: Set[String] =
+    scalacOptionsPurePrefixes ++ Set("-help", "-Xshow-phases", "-Vphases")
+
   private val scalacOptionsArgument: Argument[List[String]] =
     new Argument[List[String]] {
 
-      val underlying = StandardArgument[List[String]](scalacOptionsArg)
+      val underlying: StandardArgument[List[String]] = StandardArgument(scalacOptionsArg)
```

Comment:
Do we really need to add that type ascription? The type was pretty clear (it appears explicitly) in the former code.

Reply:
<img width="620" alt="image" src="https://user-images.githubusercontent.com/18601388/171147060-0a10adeb-b68a-4890-8209-9015257d3812.png">

We don't really need to, but IntelliJ warns you about it by default.
IMO the codebase is more welcoming for new contributors if we keep the yellow highlights in their IDEs to the bare minimum.

Reply:
That warning looks spurious to me, that field isn't public (it's an anonymous class, so it doesn't appear in the public API of this module).

Reply:
I don't think IntelliJ is smart enough to notice the difference 🧠
Still, doesn't hurt us to keep the type explicit here, I think